helm/cassandra-operator/crds/db.orange.com_cassandrabackups_crd.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
why is helm changing here ? It should be updated in that PR. Let's have only the kuttl tests and nothing else

Reply:
Reverted :)